src/lib/ext-activitylog.js

```diff
@@ -183,6 +184,17 @@ class View {
     this.apiNameFilter.updateFilterCheckboxes(filteredLogs);
   }
 
+  setFilterOptionsFromURL(updateFilter) {
+    const { id, viewType, type, name, keyword, timeStamp } = updateFilter;
+
+    this.extFilter.setFilterFromURL(id);
```

Comment:
`setFilterOptionsFromURL` and `setFilterFromURL` are independent of the URL. For example, if we ever store the filter in a file in the future, the same logic can be used. Please choose a meaningful than that is generic enough to serve the current use case and this potential future use case.

Reply:
I renamed it to `setExistedFilters` since these filters were set before.